src/attestations.ts

```
@@ -42,14 +44,40 @@ export class Client {
+      }
+
+      if (response.result?.attestations) {
+        result = result.concat(response.result.attestations);
+      }
+
+      // paging
+      const linkHeader =
+        typeof response.headers.link === "string"
+          ? response.headers.link
+          : response.headers.link?.join(", ");
+      pagesRemaining = !!(linkHeader && linkHeader.includes('rel="next"'));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)